reactioncommerce/reaction
# [Marketplace] Inviting to groups should be scoped to user's roles + UI change

Resolves #2189 

**NB:** This PR:
* Changes to ensure Admin Invite can only be done into groups that an admin performing the invite has permissions for. These changes are done both in the methods and in the UI components
* Ensures that owner has all roles that are in other groups (plus the "owner" role). I found that roles such as "index" was not in the owner's list. We were giving owner general access based on the presence "owner" role alone.
* Includes some UI improvements (a little step closer to the zeplin mocks). 
* Freezes edits on Owner permissions list
* Adds pop-up confirmation before removing a user from a group

**How To Test:**
* As the shop owner, create a new group (sample name: Accounts Manager). Toggle on the `accounts` permission for that group.
* Invite a user to that group
* Login as the newly invited user, and confirm that he can only invite to that group (or any other group that have that user have it's permissions). 
* In contrast, a shop manager, for example, has more roles and can invite into that new "Accounts Manager" group.
